src/gui/presets.cpp

```diff
@@ -1259,6 +1276,14 @@ void FurnaceGUI::initSystemPresets() {
       )
     }
   );
+  ENTRY(
+  "Sord M5", {
+    CH(DIV_SYSTEM_SMS, 1.0f, 0,
+      "customClock=1770000\n"
```

Comment:
Please confirm whether this is correct. I need *exact* rate.

I have a feeling this is actually `PAL*2/5`...

Reply:
Its correct clock rate is 1,770,000 Hz, consider this is calculated as PAL*2/5 for sure. Converted to the clock rate which is 0.4 MHz. I still wouldn't confirm it though.

Reply:
`PAL*2/5` actually is 177**3447.5**Hz though.

Reply:
Let's not do .5 as Furnace doesn't accept it.